Calendar/BookingsDiagnosticSummary.ps1

```diff
@@ -0,0 +1,167 @@
+
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Version 0.00.00.1
+
+#
+# .DESCRIPTION
+# This Exchange Online script runs the Get-BookingsDiagnosticSummary script and returns a summary of basic bookings tests on
+# a selected Bookings mailbox. The script will return the Bookings Mailbox, Staff, Services, Availability, Custom Questions,
+# as well as MessageTrace logs for the past 5 days. The script will also return the Staff Membership Log for the Bookings mailbox.
+#
+# .PARAMETER Identity
+# The Bookings mailbox SMTP address to query.
+#
+# .PARAMETER Staff
+# Verify Staff permissions for the Bookings mailbox.
+#
+# .PARAMETER StaffMembershipLog
+# Get the Staff Membership Log for the Bookings mailbox.
+#
+# .PARAMETER Graph
+# Use Graph API to get the Bookings mailbox, Staff, Services and Availiability.
+#
+# .PARAMETER GetMessageTrace
+# Get MessageTrace logs for the Bookings mailbox(Past 5 days).
+#
+# .PARAMETER ExportExcel
+# Export all data to Excel.
+#
+# .EXAMPLE
+# Get-BookingsDiagnosticSummary.ps1 -Identity fooBooking@microsoft.com
+#
+# Get-BookingsDiagnosticSummary.ps1 -Identity fooBooking@microsoft.com -Staff
+#
+# Get-BookingsDiagnosticSummary.ps1 -Identity fooBooking@microsoft.com -StaffMembershipLog
+#
+# Get-BookingsDiagnosticSummary.ps1 -Identity fooBooking@microsoft.com -Graph
+#
+# Get-BookingsDiagnosticSummary.ps1 -Identity fooBooking@microsoft.com -GetMessageTrace
+#
+# Get-BookingsDiagnosticSummary.ps1 -Identity fooBooking@microsoft.com -ExportExcel
+#
+param
+(
+    [Parameter(Position=0, Mandatory=$False, HelpMessage="Specifies the Bookings mailbox to be accessed.")]
+    [ValidateNotNullOrEmpty()]
+    [string]$identity,
+
+    [Parameter(Position=1, Mandatory=$False, HelpMessage="Verify Staff permissions for the Bookings mailbox.")]
+    [bool]$Staff=$true,
```

Comment:
>$Staff [](http://example.com/codeflow?start=10&length=6)

Staff is not clear
VerifyStaffPerms is more specific on what it does.

Any reason why they would not do this?